robot/basestation/static/js/elements/feed/navbar.js

```diff
@@ -37,10 +37,10 @@ $(document).ready(() => {
     let url = window.location.pathname
     let toggleButtonID = 'toggle-'
 
-    if (url == '/rover') {
+    if (url == '/') {
       listener = ROVER_LISTENER_TASK
       toggleButtonID += 'rover-listener-btn'
-    } else if (url == '/' || url == '/arm') {
+    } else if (url == '/arm') {
       listener = ARM_LISTENER_TASK
```

Comment:
do we need these changes lol? they seem unrelated

Reply:
I guess you want to sync master with comms-master(?)

Reply:
I merged master into here
I guess that might have been a mistake lel